main.js

```diff
@@ -11,3 +20,25 @@ function addEvent() {
   scheduleText.textContent = 'Scheduled Events For ' + event.target.textContent
   console.log(event.target)
 }
+
+addEntryButton.addEventListener('click', modalAppear)
+
+function modalAppear() {
+  modal.style.display = 'initial'
+}
+
+submitButton.addEventListener('click', addText)
```

Comment:
Instead of listening for a click event on the submit event you should be listening for a submit event on the form element
You will then need to learn about this method to keep the page from refreshing when the form is submit
https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault